cmd/agent/app/httpserver/collector_proxy.go

```diff
@@ -28,29 +27,15 @@ import (
 type collectorProxy struct {
 	samplingClient sampling.TChanSamplingManager
 	baggageClient  baggage.TChanBaggageRestrictionManager
```

Comment:
I am confused a bit. The `reporter.WrapWithMetrics` makes total sense - a metrics functionality in the general reporter package. But the config management is under `httpserver` (?), not in its own separate `clientcfgmgr` package with tchannel/grpc sub-packages. Is this just legacy setup?

Reply:
just a legacy stuff we can refactor once this is done. The grpc manager is in reporter/grpc package not ideal..